packages/@okta/vuepress-site/docs/guides/submit-app/create-guide/index.md

```diff
@@ -0,0 +1,73 @@
+---
+title: Prepare a customer-facing configuration guide
+---
+
+A configuration guide helps your customers understand how to configure your app in Okta to meet their needs.
+
+You need to provide a configuration guide as part of the OIN submission process. Your guide is provided to administrators through the Okta Admin Console. Okta checks your document for general adherence to the configuration guide instructions. After your app is published to the OIN, you can make the link public or customer‐accessible.
+
+You can create the guide in whatever format works best for you: a Web page, a Google or Word doc, or a PDF are all acceptable.
+
+## Examples
+
+Some examples of detailed configuration guides:
+
+SAML:
+
+* [GitHub Enterprise](https://saml-doc.okta.com/SAML_Docs/How-to-Configure-SAML-2.0-for-Github_Enterprise.html)
+* [Runscope](https://saml-doc.okta.com/SAML_Docs/How-to-Configure-SAML-2.0-for-Runscope.html)
+* [Salesforce](https://saml-doc.okta.com/SAML_Docs/How-to-Configure-SAML-2.0-in-Salesforce.html)
+* [Zoom.us](https://saml-doc.okta.com/SAML_Docs/How-to-Configure-SAML-2.0-for-Zoom.us.html)
+
+SCIM:
+
+* [Atlassian Cloud](https://confluence.atlassian.com/cloud/configure-user-provisioning-with-okta-957492956.html)
+* [Fuze](http://images.em.fuze.com/Web/ThinkingPhones/%7Be980dc53-8c7e-4758-b5e5-3fa20845c561%7D_Fuze_Admin_Guide_-_Okta_Provisioning_(1).pdf) (PDF link)
+* [Zscaler](https://help.zscaler.com/zia/saml-scim-configuration-guide-okta)
+
+## Content
+
+Your configuration guide should include the following sections:
+
+### Prerequisites
+
+In this section, specify any prerequisites required before the customer configures your provisioning integration in Okta. Examples may include enabling specific Okta features or SKUs, enabling API access to your SCIM server, or adding a particular version of an app in Okta.
+
+For example:
+
+`When using SAML as the SSO mode with provisioning, a specific account plan needs to be enabled on the application side for silent activation.`
```

Comment:
Not sure about using back-ticks to mark up these text examples, since they're not code or references to names or values in code. What do you think about using a regular paragraph with quotation marks?

Reply:
I changed them into the indented format (the ">" character) that gives them separation.